src/libraries/Microsoft.Extensions.Primitives/src/Microsoft.Extensions.Primitives.ref.g.cs

```diff
@@ -151,36 +156,37 @@ public partial struct Enumerator : System.Collections.Generic.IEnumerator<Micros
         public static bool operator !=(Microsoft.Extensions.Primitives.StringValues left, object? right) { throw null; }
         public static bool operator !=(Microsoft.Extensions.Primitives.StringValues left, string? right) { throw null; }
         public static bool operator !=(Microsoft.Extensions.Primitives.StringValues left, string?[]? right) { throw null; }
         public static bool operator !=(object? left, Microsoft.Extensions.Primitives.StringValues right) { throw null; }
-        public static bool operator !=(string? left, Microsoft.Extensions.Primitives.StringValues right) { throw null; }
+        public static bool operator !=(string left, Microsoft.Extensions.Primitives.StringValues right) { throw null; }
         public static bool operator !=(string?[]? left, Microsoft.Extensions.Primitives.StringValues right) { throw null; }
-        void System.Collections.Generic.ICollection<System.String?>.Add(string? item) { }
-        void System.Collections.Generic.ICollection<System.String?>.Clear() { }
-        bool System.Collections.Generic.ICollection<System.String?>.Contains(string? item) { throw null; }
-        void System.Collections.Generic.ICollection<System.String?>.CopyTo(string?[] array, int arrayIndex) { }
-        bool System.Collections.Generic.ICollection<System.String?>.Remove(string? item) { throw null; }
-        System.Collections.Generic.IEnumerator<string?> System.Collections.Generic.IEnumerable<System.String?>.GetEnumerator() { throw null; }
-        int System.Collections.Generic.IList<System.String?>.IndexOf(string? item) { throw null; }
-        void System.Collections.Generic.IList<System.String?>.Insert(int index, string? item) { }
-        void System.Collections.Generic.IList<System.String?>.RemoveAt(int index) { }
-        System.Collections.IEnumerator System.Collections.IEnumerable.GetEnumerator() { throw null; }
+        void System.Collections.Generic.ICollection<string?>.Add(string? item) { }
+        void System.Collections.Generic.ICollection<string?>.Clear() { }
+        bool System.Collections.Generic.ICollection<string?>.Contains(string? item) { throw null; }
+        void System.Collections.Generic.ICollection<string?>.CopyTo(string?[]? array, int arrayIndex) { }
+        bool System.Collections.Generic.ICollection<string?>.Remove(string? item) { throw null; }
+        System.Collections.Generic.IEnumerator<string?>? System.Collections.Generic.IEnumerable<string?>.GetEnumerator() { throw null; }
+        int System.Collections.Generic.IList<string?>.IndexOf(string? item) { throw null; }
+        void System.Collections.Generic.IList<string?>.Insert(int index, string? item) { }
+        void System.Collections.Generic.IList<string?>.RemoveAt(int index) { }
+        System.Collections.IEnumerator? System.Collections.IEnumerable.GetEnumerator() { throw null; }
```

Comment:
nullability change, cc @eerhardt

Reply:
This looks like a bug in the tool you are using to generate the file. In https://raw.githubusercontent.com/dotnet/runtime/main/src/libraries/System.Runtime/ref/System.Runtime.cs, `IEnumerable` is declared as:

```C#
    public partial interface IEnumerable
    {
        System.Collections.IEnumerator GetEnumerator();
    }
```